custom_model_runner/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+#### [1.11.0] - 2023-06-30
```

Comment:
```suggestion
#### [1.11.0] - 2024-05-01
```

Reply:
I also noticed there are git-tags for versions. Do I need to apply that manually or will it get added automatically when I merge this PR?